src/variants/beamos/config

```diff
@@ -10,7 +10,7 @@ export MODULES="base(raspicam, network, disable-services(dev-libs(octopi, beamos
 # export MODULES="base(raspicam, network, disable-services(dev-libs(octopi, beamos)), password-for-sudo)"
 
 # Proxy server
-#export BASE_PYPI_INDEX="http://devpi.soll.xyz:5798/mrbeam/prod/+simple/"
+export BASE_PYPI_INDEX="http://datamania.atbo.studio:3141/root/dev/"
```

Comment:
who's is this website from? 😕

Reply:
RIP my devpi, long live Josef's devpi :D

Reply:
For info the devpi is a cache server for python packages.

It allows to deploy pre built binaries for packages that otherwise require to be built from scratch, such as numpy on arm7l architecture.

https://www.devpi.net/

Reply:
On a completely unrelated note, i found this tool which can pre-approve all ssh keys for mrbeam devs by simply contacting github to get a public key signature.

https://www.systutorials.com/docs/linux/man/1-ssh-import-id-gh/

Reply:
That also means the **USB support key can automatically import those keys from github**. 

Any other company ssh keys can be deprecated because mrbeam developers that needs to access the device will be able to do so by simply plugging in the support key.

Lastly, **the ssh server could be enabled only for the duration of the key being inserted**. In the pre- and post-processing, you can `systemctl enable ssh` and `systemctl disable ssh`

That way all ssh attack surface is gone, and you only have to maintain gpg key infra for the usb keys.